util/plancodec/codec.go

```diff
@@ -65,7 +65,12 @@ var decoderPool = sync.Pool{
 }
 
 // DecodePlan use to decode the string to plan tree.
-func DecodePlan(planString string) (string, error) {
+func DecodePlan(planString string) (res string, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = errors.New("DecodePlan panicked")
```

Comment:
We need to print the panic stack to show how this panic happens.

Reply:
debug.PrintStack()

Reply:
zap.Any("recover", r)

Reply:
Updated. PTAL~